docs/metrics/metrics-operators/quantize.md

```diff
@@ -11,14 +11,18 @@ You can specify:
 * The size of the time buckets across which Sumo aggregates your metrics. If you do not specify a quantization interval, Sumo determines an optimum size for time buckets, as described in [Automatic quantization at query time](../introduction/metric-quantization.md).  
 * The rollup type that Sumo uses to aggregate the individual data points in a time bucket, which can be one of `avg, min, max, sum,` or `count`. If you do not specify a rollup type in the `quantize` clause of your query, for each time bucket, Sumo presents the average of the data points in that bucket.  
 
-## quantize syntax
+## Syntax
 
-:::note
-The `quantize` operator *must* appear immediatdly after your query selector, before any other operators.
```

Comment:
Most likely, this was just an (over)simplification. To apply the quantization directly to the selector, `quantize` has to follow the selector immediately. Otherwise, it will just work on top of whatever has been returned by the previous step.

Reply:
I added your comment as a note 👍